### ISSUE TITLE
ReactPlaceholder text component doesn't override the width property

### ISSUE THREAD

**Issue by awnigharbia** [buildo/react-placeholder#78]
The `Text` component doesn't let you override the `width` property and it always makes it `width:100%` 
```js
 <ReactPlaceholder
            key="placeholder"
            type="text"
            showLoadingAnimation={true}
            ready={false}
            rows={6}
            style={{ width: "70%" }}
            color="#E0E0E0"
          >
```
This doesn't change the `width` to `70%` and it always makes it `100%`.

**Comment by WandersonAlves** [comment]
+1

**Comment by WandersonAlves** [comment]
Minor workaround:

```
public componentDidMount () {
    const el: any = document.getElementsByClassName('text-block show-loading-animation')[0];
    if (el) {
      el.style.width = 'unset';
    }
  }
```

**Comment by FrancescoCioria** [comment]
released in `3.0.2`
